File: test-suite-docs-java/src/test/java/io/micronaut/discovery/docs/HelloClient.java
package io.micronaut.discovery.docs;

// tag::imports[]
import io.micronaut.http.annotation.Get;
import io.micronaut.http.client.annotation.Client;
// end::imports[]

// tag::class[]
@Client(id = "hello-world")
public interface HelloClient {

    @Get("/hello")
    String hello();
}
// end::class[]

File: test-suite-docs-kotlin/src/test/kotlin/io/micronaut/discovery/docs/HelloClient.kt
package io.micronaut.discovery.docs

// tag::imports[]
import io.micronaut.http.annotation.Get
import io.micronaut.http.client.annotation.Client
// end::imports[]

// tag::class[]
@Client(id = "hello-world")
interface HelloClient {

    @Get("/hello")
    fun hello(): String
}
// end::class[]
